.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [ '1.20', '1.21' ]
+        go-version: [ '1.21' ]
```

Comment:
Optionally, we can have `[ '1.21', '1.22' ]` here 🤷

Reply:
I don't see a reason to :sweat_smile: